composer.json

```diff
@@ -52,5 +56,10 @@
         "branch-alias": {
             "dev-master": "2.0-dev"
         }
+    },
+    "scripts": {
```

Comment:
We're using `make` to handle these.
I'm not opposed to adding the scripts but let's please invoke the appropriate make targets

Reply:
See, I'm running on Windows and the composer scripts allow me to run things without `make`.
What exactly would the make targets/parameters translate to in composer context?

Reply:
The problem is that we'll now have two places to maintain things.

Make gives us things like defining dependencies and parallelism that composer doesn't, and I don't want migrate away from it.

I know it's inconvenient to install gnu make on windows, but I prefer not to introduce the scripts if we're only repeating the definitions.

Reply:
I've added only the basic script calls so that only little maintenance is required, and that they can be run each using the options defined in their respective configuration files. I'd argue that if a toolset changes, it's a non-issue to add/remove it here too.

If one needs to add extra command line options, [they can indeed do this via composer](https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands) and do so from within their favourite IDE:
![composer-script-settings](https://github.com/nikic/FastRoute/assets/592497/d11e81ff-f333-4e17-93b5-c8fca722074d)

I find it rather unusual to use `make` in a PHP project for tasks that can be done with the built-in tools, in fact, I can't recall seeing this in any other project I`ve used or contributed before.